v2/tools/generator/internal/testcases/property_assignment_test_case.go

```diff
@@ -279,11 +280,15 @@ func (p *PropertyAssignmentTestCase) createTestMethod(
 		dst.NewIdent(errId),
 		astbuilder.CallQualifiedFunc("err", "Error"))
 
+	// match := cmp.Equal(subject, actual, cmpopts.EquateEmpty())
+	// We include cmpopts.EquateEmpty() to allow empty slices and maps to match nil values
```

Comment:
Same question: Is this change right? Looks like change from a different PR you already merged?